src/Conversions.ts

```diff
@@ -18,15 +18,20 @@ const relayPricer = new RelayPricer();
  * @returns BigNumber
  */
 export const getPrecision = (
-    precision: BigNumber | string | number = RBTC_CHAIN_DECIMALS,
+    precision: InputParam = RBTC_CHAIN_DECIMALS,
     base = 10
 ): BigNumber => new BigNumber(base).pow(precision);
 
+/**
+ * Input param that can be converted to BigNumber
+ */
+type InputParam = BigNumber | string | number;
```

Comment:
not very clear name. Also you should be able to use `BigNumberish` instead, right?

Reply:
updated